projects/packages/waf/actions.php

```diff
@@ -12,6 +12,10 @@
 	return;
 }
 
+if ( ! Waf_Runner::is_enabled() ) {
```

Comment:
Not tested, but doesn't this reintroduce the bug where the API endpoints aren't available unless it's enabled, but we need the endpoints for the UI to work to begin with? We may need a wpcom-specific guard instead; ask Crew for suggestions?

Reply:
That's a good point, I'll try to think of something else and if I can't find any solution I'll ask Crew for a suggestion, thanks for catching this!